loadtest/publish.py

```diff
@@ -0,0 +1,193 @@
+""" Molotov-based test.
+
+This load test will generate some loads against the publishing and deleting
+endpoints of the profiler server.
+"""
+import json
+import os
+import base64
+from molotov import scenario, setup, global_setup, teardown, global_teardown
+from molotov import set_var, get_var
+
+_API = None
+
+# This is the server endpoint to run the load tests against.
+# Please uncomment the right line before running this file or define it with
+# the environment variable API_ENDPOINT.
+# We don't specify a default server so that the user has to consciously decide
+# the server to hit.
+
+# This is your local server.
+# _API = 'http://localhost:5252'
+
+# This is the staging server.
+# _API = 'https://dev.firefoxprofiler.nonprod.cloudops.mozgcp.net'
+
+# This is the various file sizes we'll generate in the global setup.
+_FILE_SIZES = (512, 1024, 5 * 1024, 20 * 1024)
+
+
+def setup_api_endpoint():
+    """Sets up the _API global that we use in all scenarii.
+    """
+    global _API
+
+    endpoint_from_env = os.getenv('API_ENDPOINT')
+    if endpoint_from_env is not None:
+        # The environment always wins over what's defined in the file.
+        _API = endpoint_from_env
+
+    if _API is None:
+        raise ValueError(
+                'Please define one API endpoint before running the test file, '
+                'either uncommenting the right line in the file or specifying '
+                'the env variable API_ENDPOINT.'
+                )
+    print(f"We'll run tests against the endpoint {_API}.")
```

Comment:
I wondered if I should share this in a common module. Do you think this would be useful? It would make it necessary to expose the API from the module instead of using the global like we do. I thought it would be a bit too complex for these 2 files, but maybe it would be nice to have a ready framework for the future... Tell me what you think!